textile/fabric_pretreatment/doctype/pretreatment_order/pretreatment_order.json

```diff
@@ -672,13 +672,18 @@
  "index_web_pages_for_search": 1,
  "is_submittable": 1,
  "links": [],
- "modified": "2024-01-03 12:43:32.738554",
+ "modified": "2024-03-13 12:54:22.169085",
  "modified_by": "Administrator",
  "module": "Fabric Pretreatment",
  "name": "Pretreatment Order",
```

Comment:
Lets remove "Manufacturing User" and "Sales User (Read Only)" from Pretreatment Order